docs/src/content/docs/changelog.mdx

```diff
@@ -28,7 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## v3.0.0-alpha.10 - 2025-07-06
 
 ### Breaking Changes
-
+- `app.Env.GetAccentColor` to get the accent color of a user's system. Works on macOS. by [@etesam913](https://github.com/etesam913)
```

Comment:
Needs moving to Unreleased section - still trying to work out how we can make this better.